cross-cloud.yml

```diff
@@ -158,7 +158,7 @@ projects:
     display_name: Linkerd
     sub_title: Service Mesh
     gitlab_name: linkerd
-    project_url: "https://github.com/linkerd/linkerd"
+    projec9_url: "https://github.com/linkerd/linkerd"
```

Comment:
projec9_url should be project_url

Reply:
Odd, not even sure how that got mangled. Good catch.